src/lib/y2storage/autoinst_profile/partition_section.rb

```diff
@@ -124,6 +124,7 @@ def self.attributes
       #   @return [Integer] the partition number of this partition
 
       # @!attribute partition_type
+      #   TODO: change documentation
```

Comment:
@imobachgs, 

Can I remove the "undocumented attribute" text here?

Reply:
Yes, please. Actually it is not undocumented anymore.